src/os_sensor_nodelet.cpp

```diff
@@ -409,6 +414,8 @@ class OusterSensor : public OusterClientBase {
     void create_publishers(ros::NodeHandle& nh) {
         lidar_packet_pub = nh.advertise<PacketMsg>("lidar_packets", 1280);
         imu_packet_pub = nh.advertise<PacketMsg>("imu_packets", 100);
+        config_pub = nh.advertise<std_msgs::String>("config", 1, true);
+        config_pub.publish(config_msg);
```

Comment:
Not needed, if you follow my advice about moving the `metadata_pub` then we don't need this step since the block you added within `update_config_and_metadata` will publish the metadata as soon as we get results from the sensor. 
```suggestion
        config_pub.publish(config_msg);
```